README.md

```diff
@@ -7,7 +7,16 @@
 [![NPM](https://nodei.co/npm/simpledatevalidator.png)](https://www.npmjs.com/package/simpledatevalidator/)
 
 ## How to use?
-Pass date and date format to isValid function. Function will validate and return true/false;
+Pass date and date format to `isValidDate` function.
+Function will validate and return true or false.
+
+```
+// Create Instance of the DateValidator
+var DateValidator = require('./index');
+
+// Pass in date and format to be validated and receive a boolean
+DateValidator.isValidDate("2017-09-27","YYYY-MM-DD");
+```
```

Comment:
Here we should add the result of the function.
So something like this
`// This will return true.`

Also we would want a wrong date to validate example which will return false, which should be something like this

`DateValidator.isValidDate("27-09-27","YYYY-MM-DD");
// This will return false`

Reply:
Got it!